docs/content/latest/deploy/kubernetes/clients.md

```diff
@@ -143,9 +145,11 @@ spec:
       defaultMode: 256
 ```
 
-Here is an example of a client that uses the `YSQL shell` ([`ysqlsh`](../../../admin/ysqlsh)) to connect.
+The following example shows a client that uses the `YSQL shell` ([`ysqlsh`](../../../admin/ysqlsh)) to connect.
+
+Info pending
```

Comment:
What does the "info pending" refer to?

Reply:
I think the "here is an example" just refers to the connection command that follows.